src/main/java/com/gtnewhorizons/obama/main/tileentities/multi/instances/Obama_MetaTileEntity_Large_Assembler.java

```diff
@@ -31,6 +31,12 @@ public Obama_MetaTileEntity_Large_Assembler(String aName) {
         super(aName);
     }
 
+    @Override
+    public boolean checkMachine(IGregTechTileEntity aBaseMetaTileEntity, ItemStack aStack) {
```

Comment:
Not sure why these are needed, when we've got the super method.  I'd only expect these to be overridden if they diverge from super handling?

Reply:
the checkMachine i overrided is from its superclass but the checkMachine that i want to use is in its interface. so the superclass can't handle it because the needed interface isn't accessible for it.